packages/async-flow/test/bad-host.test.js

```diff
@@ -103,6 +111,8 @@ const testBadHostFirstPlay = async (t, zone) => {
     // Notice that the bad call was not recorded in the log
   ]);
   t.log('badHost firstPlay done');
+  // Allow panicHandler to be called.
+  await null;
```

Comment:
I'm tempted to say we should `await eventLoopIteration()` (aka `await nextCrank()`) here